7주차/[Lv.2]_가장_큰_정사각형_찾기/안혜수.js

```diff
@@ -0,0 +1,14 @@
+function solution(board) {
+  let Max = 0;
```

Comment:
```suggestion
  let max = 0;
```

`Max`를 보니 `Math.max()`가 생각이 나는 것 같아서 `max`라고 변수명을 짓는 것이 좋을 것 같습니다!! 🤔